src/configuration/catalog/catalog.md

```diff
@@ -214,19 +213,20 @@ Magento supports the following versions of Elasticsearch: 6.0+, 5.0+, and 2.0+ (
 |Elasticsearch Server Timeout|Global|Determines the number of seconds before the server times out. Default value: 15|
 |Test Connection||Validates the Elasticsearch connection.|
-|Enable Search Suggestions|Store View|Determines if search suggestions appear for common misspellings. When enabled, search suggestions are offered for any request that returns no results. Search suggestions can impact the performance of search. Options: Yes / No. When set to Yes, additional options display for Enable Search Recommendations and associated fields.|
+|Enable Search Suggestions|Store View|Determines if search suggestions appear for common misspellings. When enabled, search suggestions are offered for any request that returns no results and appear under the `Did you mean` section on the **Search results** page. Search suggestions can impact the performance of search. Options: Yes / No. When set to Yes, additional options display for Enable Search Recommendations and associated fields.|
 |Search Suggestions Count|Store View|Determines the number of search suggestions that are offered. For example: 2|
 |Show Results Count for Each Suggestion|Store View|Determines if the number of search results is shown for each suggestion. Depending on theme, the number usually appears in brackets after the suggestion. Options: Yes / No|
-|Enable Search Recommendations|Store View|Determines if search recommendations are enabled for the store view. Options: Yes / No <br/>When set to Yes, additional options display for Search Recommendations Count and Shows Results Count for Each Recommendation.|
+|Enable Search Recommendations|Store View|Determines if search recommendations are offered when a search returns no results and appear under the `Related search terms` section on the **Search results** page. Options: Yes / No <br/>When set to Yes, additional options display for Search Recommendations Count and Shows Results Count for Each Recommendation.|
 |Search Recommendations  Count|Store View|Specifies the number of  search terms offered as recommendations. By default, no more than five are shown.|
 |Show Results Count for Each Recommendation|Store View|When set to "Yes", the number of products found for the proposed search recommendation is shown in the brackets. Options: Yes / No|
+|Minimum Terms to Match|Store View|Specifies a value (percent, negative or positive integer, percent plus integer) that corresponds to the number of terms from your query that the search results should match in order to be returned. This ensures optimal results relevancy for shoppers. For example, an integer value of `3` will require that the returned results must match `3` terms specified in your query. A percent value of `50%` requires that the returned results must match half of the terms specified in your query. To learn more, see [minimum_should_match parameter](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-minimum-should-match.html).|
```

Comment:
percentage can be negative too, and negative percent can be included in the percent+integer format. not sure we need to explicitly call this out though, up to you. 

I was also under the impression that we would want to have our "own" copy of the syntax descriptions (from the ES link). I'm ok with linking to their content, but I suppose there is a small chance of the link becoming dead or broken.